src/routes/docs/advanced/self-hosting/sites/+page.markdoc

```diff
@@ -0,0 +1,131 @@
+---
+layout: article
+title: Sites
+description: Harness the full power of self-hosted sites with Appwrite. Explore site deployment, management, and integration in your self-hosted environment.
+---
+
+{% partial file="configure-github-app.md" /%}
+
+# Configure Sites runtimes {% #sites %}
+
+Not all site runtimes are enabled by default. Enable the runtimes that you need and disable unused runtimes to save disk space on your server.
+To enable a runtime, add it to the `_APP_SITES_RUNTIMES` environment variable as a comma-separated list.
+
+The example below would enable Static, Node.js 22, and Flutter 3.29 runtimes.
+
+```bash
+_APP_SITES_RUNTIMES=static-1,node-22,flutter-3.29
+```
+
+You can also configure the maximum timeout that can be set on individual Appwrite Sites. The maximum configurable timeout can be increased by changing the `_APP_SITES_TIMEOUT` environment variable. This environment variable changes the configurable maximum but does not alter existing configurations of individual sites.
+
+{% partial file="update-variables.md" /%}
+
+# Add a root domain for Sites {% #root-domain %}
+
+Appwrite allows you to use a root domain as a custom domain for your sites instead of the default Appwrite domain or a subdomain of your custom domain, if you prefer that.
+To add a root domain, you must first configure either one of the following environment variables in your `.env` file:
+
+- `_APP_DOMAIN_TARGET_A`: Set this to the IPv4 address of your server.
+- `_APP_DOMAIN_TARGET_AAAA`: Set this to the IPv6 address of your server.
+
+Ensure that the `_APP_DOMAIN_SITES` environment variable is set to your root domain (e.g., `myapp.com`).
+
+Next, head to your DNS provider and create an A or AAAA record for your root domain pointing to the server's IP address. This will allow Appwrite to serve your sites from the root domain.
```

Comment:
I'm a little confused about the usage of "root domain" here. This makes it sound like you want to point `myapp.com` to (a) Sites, however I was under the impression the `_APP_DOMAIN_SITES` env var was for pointing all subdomains to your appwrite instance so that your site can be served.

Reply:
Yeah, I was just testing this and I messed it up. I've rewritten it